publication-rest/src/main/java/no/unit/nva/publication/update/PublishingRequestResolver.java

```diff
@@ -109,9 +112,13 @@ private void autoCompletePendingPublishingRequestsIfNeeded(
                 ticket -> ticket.complete(publication, getUsername()).persistUpdate(ticketService));
     }
 
-    private boolean thereAreNoFiles(Publication publicationUpdate) {
+    private boolean thereAreNoPendingFiles(Publication publicationUpdate) {
         return publicationUpdate.getAssociatedArtifacts().stream()
-                .noneMatch(File.class::isInstance);
+                .noneMatch(this::isPendingFile);
+    }
```

Comment:
Completing ticket not only when there are no files, but also when no pending files.